src/frontend/classic/components/MediaSelect.svelte

```diff
@@ -6,6 +6,7 @@
         InlineLoading,
     } from "carbon-components-svelte";
     import PlugFilled16 from "carbon-icons-svelte/lib/PlugFilled16";
+    var Fuse = require("fuse.js");
```

Comment:
Use es `import` instead.

Reply:
Thank you for your kind suggestion, the proposed change has been applied at https://github.com/manga-download/haruneko/pull/37/commits/71a38ab46e4e33134c5ccf8766df3b3d90bfe9d1